src/ticket_activities/ticket_activities.controller.ts

```diff
@@ -5,35 +5,97 @@ import {
   Get,
   Param,
   Patch,
-  Delete,
+  UseGuards,
 } from '@nestjs/common';
 import { ActivitiesService } from './ticket_activities.service';
```

Comment:
empty line between local and global imports

Reply:
done